src/index.html

```diff
@@ -6,13 +6,63 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Link to the SCSS file is incorrect. SCSS is a preprocessor and needs to be compiled to CSS. The browser cannot interpret SCSS files directly. You should link to a `.css` file instead.